### DISCUSSION TITLE
學生認證問題

### DISCUSSION THREAD

**Discussion by pinguinjay** [community/community#63235] [GitHub Education]
### Hi everyone,

我的學校是用中華民國年分紀年而非西元紀年
而且學生證上並沒有入學與畢業年分
我已經把入學與畢業年分印出來放在學生證上
卻還是無法通過
我覺得審核上需要有人工審核
否則對於非英文語系的學生很不公平
My school uses the year of the Republic of China instead of AD
And there is no enrollment and graduation year on the student card
I have printed out the year of admission and graduation and put it on my student card
but still can't pass
I think the review needs manual review
Otherwise it is very unfair to the students of 
![螢幕擷取畫面 2023-08-09 161918](https://github.com/community/community/assets/83256594/655769cb-6db2-4351-8083-74485d6f498e)
non-English language department

**Comment by B1uePumpkin** [comment]
我也卡爛，同求解法TAT

**Comment by aprylewu** [comment]
可能可以翻译一下吧，我是大陆的学生，在这里翻译了**https://github.com/Nagi-ovo/CHSI-Converter**

**Comment by sam951013** [comment]
可以在網路上抓對照表，在附上證明的地方丟圖進去

**Comment by sam951013** [comment]
我的後來有人工審核
